src/tag/tag.js

```diff
@@ -55,25 +57,27 @@ class Tag extends React.Component<PropsT, {}> {
       $kind: kind,
     };
     return (
-      <Root {...sharedProps} {...rootProps}>
+      <Root {...sharedProps} {...rootProps} $closeable={closeable}>
```

Comment:
suggest moving this to `sharedProps` above

```suggestion
      <Root {...sharedProps} {...rootProps}>
```